attribute.go

```diff
@@ -24,6 +25,29 @@ type Attribute struct {
 	Data []byte
 }
 
+// IsNested interprets the Type field of the Attribute to determine whether it's
+// a nested attribute or not. This facilitates recursive parsing of the attribute.
+// This is the leftmost bit in Type. Mutually exclusive with IsNetByteOrder().
+func (a Attribute) IsNested() bool {
+	return (a.Type & unix.NLA_F_NESTED) > 0
```

Comment:
Using `unix` outside of `*_linux.go` files means that it won't build on non-UNIX machines.  Need to make a copy of these constants locally.